src/hoplon/core.cljs

```diff
@@ -98,9 +96,7 @@
                                   (when-not (new? k)
                                     (let [n (->node k)]
                                       (when (= this (.-parentNode n))
```

Comment:
Do we still want to check for `(.-parentNode)`? I don't think it's possible to hook into the basic `removeChild`/`insertBefore`/`appendChild` and mess with the DOM while iterating.